pkg/manager/config/config.go

```diff
@@ -271,7 +282,22 @@ func (i *Instance) GetNoBrowser() bool {
 	return i.getBool(NoBrowser)
 }
 
+func (i *Instance) GetNotificationsEnabled() bool {
+	return i.getBool(NotificationsEnabled)
+}
+
+func (i *Instance) GetConfigUpdatesChannel() chan int {
```

Comment:
Is this used? The channel doesn't appear to be used at all.

Reply:
This is for updating the systray, which isn't working yet. It's a requirement for auto-update, so I've commented it out for now.